nixos/tests/openssh.nix

```diff
@@ -77,5 +84,10 @@ in {
                        " server_lazy true");
 
     };
+
+    subtest "configuration", sub {
+      $client->succeed("ssh -G other_server | grep -i 'addressfamily any'");
+      $client->succeed("ssh -G example_server | grep -i 'addressfamily inet'");
```

Comment:
Not sure if we need an extra test for that.

Reply:
Fair enough, will drop that commit.